BUILD

```diff
@@ -210,23 +210,23 @@ NODEJS_VARIATIONS = [
 
 NODEJS = {
     "{REGISTRY}/{PROJECT_ID}/nodejs" + version + "-" + distro + ":" + tag_base + "-" + arch: "//nodejs:nodejs" + version + label + "_" + user + "_" + arch + "_" + distro
-    for arch in BASE_ARCHITECTURES
-    for distro in ARCHITECTURES
+    for arch in ARCHITECTURES
+    for distro in LANGUAGE_DISTROS
     for version in NODEJS_VERSIONS
     for (tag_base, label, user) in NODEJS_VARIATIONS
 }
 
 # oci_image_index
 NODEJS = {
```

Comment:
Shouldn't this be `NODEJS |= {`?
The line 211 above already is `NODEJS = {`

Reply:
yeah probably.